src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -211,6 +261,10 @@ public async Task InitAsync()
                 });
 
                 Stats.ConnectedAt = DateTime.UtcNow;
+                
+                _actualConnectionScope = _parentConnectionScope.For(SubscriptionOperationScope.ActualConnection);
+                _parentConnectionScope.RecordConnectedAt(Stats.ConnectedAt);
```

Comment:
no need to do that - our actual connetion scope will have proper timing for that.

Reply:
done - removed the '`ConnectedAt`'